src/utils/fees.ts

```diff
@@ -67,6 +68,7 @@ export const calculatePaymentTxnFee = async (
       {
         payee,
         amount,
+        memo: memo || undefined,
```

Comment:
Can this just be 
```
    payments: [
      {
        payee,
        amount,
        memo,
      },
    ],
```
?

Reply:
Actually it needed to be `encodeMemoString(memo)` good catch haha